### ISSUE TITLE
Javascript part of the html file

### ISSUE THREAD

**Issue by vivekkumar27june88** [vim-dist/webvim#52]
Hi

Is it possible to apply javascript rule on the inlined javascript content of a html file?

Thanks

**Comment by krampstudio** [comment]
Yes, it is but when you do that you actually lose the html syntax color. For example I'm used to change the filetype when I'm editing : 
`set filetype=javascript` 

May be some Vim guru have a better solution

**Comment by krampstudio** [comment]
Looking back at this issue, here are my findings : 
 - Using regions https://vim.fandom.com/wiki/Different_syntax_highlighting_within_regions_of_a_file 
 - Some plugins helps you automatizing the region defintion : https://github.com/inkarkat/vim-SyntaxRange

I can imagine having such rules in an html files within `<script>` or `<style>` tags, but also within documentation files, like markdown...
